css/css-box/margin-trim/flex-column-inline-multiline-ref.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>margin-trim: flex-column-inline-multiline</title>
+<link rel="author" href="mailto:sammy.gill@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-box-4/#margin-trim-flex">
+<meta name="assert" content="inline-start should be trimmed for all items on the first flex line and inline-end should be trimmed for all items on the last flex line">
+<style>
+flexbox {
+    display: flex;
+    width: 60px;
```

Comment:
Is there a reason the reference case is using an explicit `width: 60px` (vs. `width: min-content` on the corresponding element in the testcase, and in most/all of the other similar test files in this patch)?

This `width` doesn't seem related to the thing being tested, so might be good to make it consistent one way or another (probably switching to `min-content` here), to avoid having unnecessary/arbitrary differences between testcase and reference case.

Reply:
Yeah I cannot remember why this was set here, but agreed that it can be changed. I changed `width` to `min-content` on the containers where I think it made sense. Let me know if you see anything else!